tasks/deployment/update-merkle-distributor-rewards-parameters.ts

```diff
@@ -0,0 +1,17 @@
+import { types } from 'hardhat/config';
+
+import mainnetAddresses from '../../src/deployed-addresses/mainnet.json';
+import { hardhatTask } from '../../src/hre';
+import { DIP_UPDATE_REWARDS_IPFS_HASH } from '../../src/lib/constants';
+import { updateMerkleDistributorRewardsParametersProposal } from '../../src/migrations/update-merkle-distributor-rewards-parameters-proposal';
+
+hardhatTask('deploy:update-merkle-distributor-rewards-parameters-proposal', 'Create proposal to wind down the borrowing pool.')
```

Comment:
Nit: Update task description (currently is `Create proposal to wind down the borrowing pool.`).